<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>均仓策略（仅支持币安/ETH-USDT现货）   联系作者：1259419892@qq.com</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Api密钥</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>401</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>key</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="keyLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>secret</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="secretLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>401</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>策略超参数</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>401</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>波动率（%）</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="volaLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>最小交易数量(&gt;0.001)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="minAmountLineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>401</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>其他设置</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>401</width>
           <height>67</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>最小交易额（usdt）</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="minTradeQuanLineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>开机自启</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>执行间隔（S）</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="intervalLineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>590</y>
          <width>401</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="startPushButton">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>610</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>启动</string>
        </property>
       </widget>
       <widget class="QPushButton" name="testConnectPushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>610</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>测试连接</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>320</y>
          <width>401</width>
          <height>201</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>策略状态</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>401</width>
           <height>171</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>USDT</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="usdtLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>ETH</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ethLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Buy</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="buyLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Sell</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="sellLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>last price</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lastLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>last trade price</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="stopPushButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>610</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>771</width>
      <height>651</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="../Resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
